### PR TITLE
Register application (checkpoint 4)

### PR DESCRIPTION
**Pull request by wolffles**
description: I added the model, controller, views, and rspec tests all for a user to register an application to his/her account. also switched to PG database.
test coverage: 96.63%

### CODE REVIEW

File: app/assets/javascripts/registered_application.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/registered_application.js]: @wolffles not using, please delete.

File: app/assets/stylesheets/registered_application.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the registered_application controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/registered_application.scss]: @wolffles not using, please delete.

File: app/helpers/registered_application_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module RegisteredApplicationHelper
```
Comment (brittanyjspringer) [inline app/helpers/registered_application_helper.rb]: @wolffles not using, please delete.

File: application.yml [hunk 4]
```diff
@@ -1,2 +0,0 @@
-SENDGRID_USERNAME: app71218754@heroku.com
```
Comment (brittanyjspringer) [inline application.yml:1]: @wolffles I hope you canceled these keys.

File: spec/models/registered_application_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/models/registered_application_spec.rb:1]: @wolffles there should be tests here.

File: spec/models/user_spec.rb [hunk 6]
```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
```
Comment (brittanyjspringer) [inline spec/models/user_spec.rb]: @wolffles you can delete this since Devise handles it.

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]
@wolffles requesting changes. Please @ me when they are complete.

**Comment by wolffles** [comment]
@wonderwoman13 do i just @ you in the comment box? also for that key,  I cant find that application.yml except for in my master branch, its not valid anymore but still cant find that file in 
 this current branch in my local repo.

**Comment by brittanyjspringer** [comment]
@wolffles you deleted the file so you're in the clear as long as the keys are no longer in use. Looks good, I'll merge. Please re-submit the checkpoint on Bloc.

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**